app/services/dttp/retrieve_account.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Dttp
+  class RetrieveAccount
+    include ServicePattern
+
+    class HttpError < StandardError; end
+
+    attr_reader :provider
+
+    ACCOUNT_ATTRIBUTE_FILTERS = %w[
```

Comment:
Naming gripe, but I would call this `ACCOUNT_ATTRIBUTE_FIELDS` rather than filters.

When I see the word `filter` I assume it will be accompanied by some kind of `where` query i.e. reducing the number of accounts returned, which is not the case here.